tests/end_to_end_integration/run_test.sh

```diff
@@ -2,14 +2,14 @@
 
 set -e
 
-if [[ $# != 2 ]]
-then
-    echo "usage: tests/end_to_end_integration_argo/run_test.sh <registry> <version>"
-    exit 1
-fi
-
 SLEEP_TIME=15
 
+function printUsage {
+    echo "usage: tests/end_to_end_integration_argo/run_test.sh [-r] <registry> <version>"
```

Comment:
[nit] I think `-r` is fine, but it would be easier to follow if you use `--runNudgeToFine=True`, you can have it default to either True or False. Then you don't have to manually change `runNudgeToFine` later in this script on line 100.

Reply:
I think it makes circleci config a bit easier to read too:
```
    parameters:
      runNudgeToFine:
        default: false
        type: boolean
```